src/chrome/content/rules/sciencedirect.com.xml

```diff
@@ -1,32 +1,54 @@
 <!--
-503 Service Unavailable:
-    highlight.sciencedirect.com
+	No working URL known:
+		cert.sciencedirect.com
+		highlight.sciencedirect.com (503)
+		logger.sciencedirect.com
 
-Invalid certificate:
-    ars.sciencedirect.com
-    binary-services.sciencedirect.com
-    technology.sciencedirect.com
-    topics.sciencedirect.com
+	Time out:
+		alpha.sciencedirect.com
 
-Mixed content:
-    attach.sciencedirect.com
+	Invalid certificate:
+    	ars.sciencedirect.com
+    	binary-services.sciencedirect.com
+		info.sciencedirect.com
```

Comment:
Over `http`? Ok, I do as well.

Unfortunately, it doesn't redirect for deep links so I can't write a simple rule. 

Deep links apparently support `https` so I may be able to write a more complex rule but I'd like to get this merged today if possible, to be included in the next release.

I will investigate and see what can be done for `info.sciencedirect.com`.

test url: https://info.sciencedirect.com/techsupport/books/catalogs/food-science-nutrition-books-catalog.pdf